projects/laji/src/app/shared-modules/document-viewer/document/document.component.ts

```diff
@@ -224,16 +224,12 @@ export class DocumentComponent implements AfterViewInit, OnChanges, OnInit, OnDe
     return this._highlight;
   }
 
-  setActive(i) {
+  setActive(i: number) {
     this.active = i;
     if (this.document && this.document.gatherings) {
       this.activeGathering = this.document.gatherings[i] || {};
     }
-    this.useWorldMap = !(
-      this.activeGathering.interpretations &&
-      this.activeGathering.interpretations.country &&
-      this.activeGathering.interpretations.country === 'http://tun.fi/ML.206'
-    );
+    this.useWorldMap = !(this.activeGathering?.interpretations?.country === 'http://tun.fi/ML.206');
```

Comment:
It's not a big deal but it looks a bit weird - could just use `!==`? Negation of a logical operation somehow adds cognitive load at least for me.